app/src/main/res/layout/movie_detail_activity_layout.xml

```diff
@@ -0,0 +1,101 @@
+            app:layout_constraintDimensionRatio="w,16:4"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:srcCompat="@tools:sample/avatars[1]" />
+
+        <ImageView
+            android:id="@+id/detail_poster_image"
+            android:layout_width="90dp"
+            android:layout_height="120dp"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="8dp"
+            app:layout_constraintDimensionRatio="h,1:1"
+            app:layout_constraintEnd_toEndOf="@+id/detail_backdrop_image"
+            app:layout_constraintHorizontal_bias="0.028"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/detail_backdrop_image"
+            tools:srcCompat="@tools:sample/avatars[2]" />
+
+        <TextView
+            android:id="@+id/detail_movie_name"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="8dp"
+            android:text="TextView"
+            android:textSize="16sp"
+            android:textStyle="bold"
+            android:typeface="normal"
+            app:layout_constraintBottom_toBottomOf="@+id/detail_poster_image"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.06"
+            app:layout_constraintStart_toEndOf="@+id/detail_poster_image"
+            app:layout_constraintTop_toTopOf="@+id/detail_poster_image"
+            app:layout_constraintVertical_bias="0.282"
+            tools:text="Movie Name" />
+
+        <TextView
+            android:id="@+id/detail_movie_genres"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:layout_marginEnd="8dp"
+            android:text="@string/movie_and_genres"
+            android:textSize="12sp"
+            android:typeface="normal"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintHorizontal_bias="0.0"
+            app:layout_constraintStart_toStartOf="@+id/detail_movie_name"
+            app:layout_constraintTop_toBottomOf="@+id/detail_movie_name" />
+
+        <TextView
+            android:id="@+id/detail_release_date"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="4dp"
+            android:layout_marginBottom="8dp"
+            android:text="Release date"
+            android:textSize="12sp"
+            android:typeface="normal"
+            app:layout_constraintBottom_toBottomOf="@+id/detail_poster_image"
+            app:layout_constraintEnd_toEndOf="@+id/detail_movie_genres"
+            app:layout_constraintHorizontal_bias="0.0"
```

Comment:
Why did you add this property? What does it do?

Reply:
I truly don´t know what it does, because I used the Layout editor to create this Layout.